Library/Homebrew/formula_installer.rb

```diff
@@ -180,6 +180,10 @@ def install
     build_bottle_preinstall if build_bottle?
 
     unless @poured_bottle
+      if formula.file_modified?
+        ohai "#{formula.full_name}.rb is modified; building from source"
+        puts "To install from a bottle instead, run with --force-bottle"
```

Comment:
I wonder if there's somewhere you can unify the printing of this message in a method somewhere (just so we don't need to manually keep the two strings in sync).